Assets/EnemyAI.cs

```diff
@@ -0,0 +1,55 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemyAI : MonoBehaviour
+{
+    public Transform chaseTarget;
+    public float moveSpeed = 5f;
+    private Rigidbody2D rb;
+    public float health;
+    public float damageOverTime = 1f;
```

Comment:
Not a giant fan of this variable name - it's not really damage over time but really an attack cooldown, right?